SSLsplit/scripts/autostart_sslsplit.sh

```diff
@@ -20,5 +20,5 @@ iptables -P OUTPUT ACCEPT
 sh /pineapple/modules/SSLsplit/rules/iptables
 
 iptables -t nat -A POSTROUTING -j MASQUERADE
-
+echo "If this file exists, it means that the script succesfully detected the log directory! > /pineapple/modules/SSLsplit/log/islogthere || mkdir /pineapple/modules/SSLsplit/log/ && echo "Directory succesfully created!" > /pineapple/modules/SSLsplit/log/islogthere && printf "WRN: Log directory at '/pineapple/modules/SSLsplit/' was not found! We just created it for you! Your welcome. :) \n" > /pineapple/modules/SSLsplit/output_${MYTIME}.log
```

Comment:
Please see my last comment in "Conversation"

Few things:  

* Spelling mistake: " Your welcome. :) \n" -> "You're"
* " /pineapple/modules/SSLsplit/output_${MYTIME}.log" -> You're not even writing to the log directory?

It is standard software development practice to test your code before pushing to your repositories